test/helper/helper.go

```diff
@@ -431,5 +433,5 @@ func GenerateRandomString(length int) (string, error) {
 	// Trim any padding characters
 	randomString = randomString[:length]
```

Comment:
`panic: runtime error: slice bounds out of range [:5] with length 4`

The base64 encoding produces less characters as is unnecessary for this task. Here's a simplified approach that doesn't panic:

```go
func GenerateRandomString(length int) string {
	charSet := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
	rand.Seed(time.Now().UnixNano())
	result := make([]byte, length)
	for i := range result {
		result[i] = charSet[rand.Intn(len(charSet))]
	}
	return string(result)
}
```